lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/ipv-core-main-journey.yaml

```diff
@@ -350,6 +352,12 @@ CRI_KBV_J2:
           targetState: PYI_CRI_ESCAPE_NO_F2F
     enhanced-verification:
       targetState: MITIGATION_02_OPTIONS
+      checkFeatureFlag:
+        f2fCimit:
```

Comment:
Do we need to add this feature flag to a list somewhere, or will it Just Work?

Reply:
We add the feature flag via the common-infra repo https://github.com/govuk-one-login/ipv-core-common-infra/pull/733/files - the state machine will look for this as a parameter and will treat it as false if not present